packages/core/src/plugins/google/google.ts

```diff
@@ -74,6 +74,7 @@ export type ChatCompletionChunk = {
   model: string;
 };
 
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Not sure if it is better to leave this to match API naming convention or rename to match Rivet/JS naming convention